pkg/controller/tuned/status.go

```diff
@@ -67,51 +66,55 @@ func (r *ReconcileTuned) syncOperatorStatus() (bool, error) {
 		return requeue, nil
 	}
 
-	_, err = r.cfgv1client.ClusterOperators().UpdateStatus(coState)
+	err = r.client.Status().Update(context.TODO(), coState)
 	if err != nil {
+		glog.Errorf("Unable to update coState")
 		return requeue, err
 	}
 
 	return requeue, nil
 }
 
 func (r *ReconcileTuned) getOrCreateOperatorStatus() (*configv1.ClusterOperator, error) {
-	var err error
-
-	clusterOperatorName := ntoconfig.OperatorName()
-
-	co := &configv1.ClusterOperator{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ClusterOperator",
-			APIVersion: "config.openshift.io/v1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: clusterOperatorName,
-		},
-	}
-
-	if r.cfgv1client == nil {
-		r.cfgv1client, err = ntoclient.GetCfgV1Client()
-		if r.cfgv1client == nil {
-			return nil, err
-		}
-	}
-
-	coGet, err := r.cfgv1client.ClusterOperators().Get(clusterOperatorName, metav1.GetOptions{})
-	if err != nil {
+	co := &configv1.ClusterOperator{ObjectMeta: metav1.ObjectMeta{Name: ntoconfig.OperatorName()}}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: co.Name}, co); err != nil {
 		if errors.IsNotFound(err) {
 			// Cluster operator not found, create it
-			co_created, err := r.cfgv1client.ClusterOperators().Create(co)
-			if err != nil {
-				// Failed to create the cluster operator object
-				return nil, err
+			initializeClusterOperator(co)
```

Comment:
Was this done before elsewhere?

Reply:
Good point.  No, this was not done before.  I don't think this is strictly necessary and I have mixed feelings about this myself, but it does initialize the ClusterOperator object rather than keeping it empty.